src/javascript/app/base/interview_popup.js

```diff
@@ -0,0 +1,37 @@
+const Cookies        = require('js-cookie');
+const BinarySocket   = require('./socket');
+const isEuCountry    = require('../common/country_base').isEuCountry;
+const Client         = require('../base/client');
+
+const InterviewPopup = (() => {
+    const onLoad = () => {
+        BinarySocket.wait('authorize', 'website_status', 'get_account_status').then(() => {
+            if (Client.isLoggedIn() && isEuCountry() && !Cookies.get('InterviewConsent')) {
+                const interview_popup = $('#interview_popup_container');
+                const interview_no_thanks = $('#interview_no_thanks');
+                const interview_ask_later = $('#interview_ask_later');
+                const interview_interested = $('#interview_interested');
```

Comment:
these are all jquery elements, so the variable name should start with `$`, i.e. `$interview_popup` as per the coding styleguide